src/main/scala/io/citrine/lolo/trees/splits/ClassificationSplitter.scala

```diff
@@ -37,7 +37,7 @@ case class ClassificationSplitter(randomizedPivotLocation: Boolean = false, rng:
 
       /* Use different spliters for each type */
       val (possibleSplit, possibleImpurity) = rep._1(index) match {
-        case _: Double => getBestRealSplit(data, calculator, index, minInstances)
+        case _: Double => Splitter.getBestRealSplit[Char](data, calculator, index, minInstances, randomizedPivotLocation, rng)
```

Comment:
Is there any symmetry in `getBestCategoricalSplit` between the different subtypes that we can shuttle off to a static method? 

For instance, it seems that `getBestCategoricalSplit` on the `MultiTaskSplitter` is the most general, since it operates on an arbitrary number of distinct label types. Could that logic be adapted to provide a general method here?

Reply:
I looked into that a bit yesterday, but the codes seemed different in a more significant way, and it was the end of the day and my motivation for refactoring things had waned. 🤷